.github/workflows/release.yml

```diff
@@ -18,27 +17,44 @@ jobs:
         with:
           python-version: '3.13'
 
-      - name: Check package version matches tag
+      # Install dependencies for Poetry and the package
+      - name: Install dependencies
         run: |
-          pkgver=$(python -c "import autocorpus; print(autocorpus.__version__, end='')")
```

Comment:
I believe the issue is that the python being used here is not the python that the package was installed in above (in the `uses: ./.github/actions/setup`) part.

So the fix should be a simple as changing this line to:
```
pkgver=$(poetry run python -c "import autocorpus; print(autocorpus.__version__, end='')")
```

Reply:
Brilliant, thanks for looking into this, I can't see anything I can do other than "resolve conversation", so I think only TB/JP/Antoine can approve the changes/PR.